docs/Formula-Cookbook.md

```diff
@@ -761,44 +761,79 @@ For example, Ruby 1.9’s gems should be installed to `var/lib/ruby/` so that ge
 
 Another example would be configuration files that should not be overwritten on package upgrades. If after installation you find that to-be-persisted configuration files are not copied but instead *symlinked* into `/usr/local/etc/` from the Cellar, this can often be rectified by passing an appropriate argument to the package’s configure script. That argument will vary depending on a given package’s configure script and/or Makefile, but one example might be: `--sysconfdir=#{etc}`
 
-### launchd plist files
+### Service files
 
-Homebrew provides two formula DSL methods for launchd plist files:
+Homebrew provides two formula DSL methods for service files:
```

Comment:
Were the two list items following meant to be removed? Otherwise this line has nothing to introduce.